experimental/packages/opentelemetry-api-metrics/src/types/BatchObserverResult.ts

```diff
@@ -14,17 +14,17 @@
  * limitations under the License.
  */
 
-import { Labels } from './Metric';
+import { Attributes } from './Metric';
```

Comment:
According to spec, the attribute interface is now shared between metric and trace. Should we just use the Attributes from the 1.0 API?

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes

Reply:
That makes sense, but I believe this should happen in a follow-up PR. I assume there will be some breaking changes, especially with https://github.com/open-telemetry/opentelemetry-js/issues/2554 and https://github.com/open-telemetry/opentelemetry-proto/pull/342 (including removing the deprecated labels field) coming up. I am not sure how to best navigate that change, as it seems quite big. I think what needs to happen for that to work is:

- Update the Proto to the latest version (as the currently referenced version does not have attributes yet)
- Either update the hand-rolled proto implementation, or set up a way of auto-generating JS code from the proto directly.
- Then we can update the actual reference of what were labels before to the new attributes.

These things are only somewhat related to the renaming of `Labels` to `Attributes`, but without updating the underlying data structures as well as the code, I am not sure we can make this change without breaking the code or tests.

Reply:
Sounds good. I'll create a follow up story for that.